OneCommunityGlobal/HGNRest
# Xiaoyu add editable container for info

# Description
![Screenshot 2023-07-19 at 6 30 58 PM](https://github.com/OneCommunityGlobal/HGNRest/assets/102696148/ac23c688-6414-4a5c-aeae-b747c12cf6e7)

## Related PRS (if any):
To test this backend PR you need to checkout the [#1025](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1025) frontend PR
…

## Main changes explained:
- Add information controller
- modifies routes
- add informationrouter
- modify server
- add information models
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
Login as Owner:
https://github.com/OneCommunityGlobal/HGNRest/assets/102696148/b64a595e-2d8e-4e11-b645-30eec40ac6e8
Login as Admin:
https://github.com/OneCommunityGlobal/HGNRest/assets/102696148/60d049a5-5014-421a-aeb9-cd3add6a16b0
Login as Volunteer:
![Screenshot 2023-07-19 at 6 21 02 PM](https://github.com/OneCommunityGlobal/HGNRest/assets/102696148/f310d23c-31a2-4d68-9f1f-3eca10960404)



## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
